pkg/components/aws-ebs-csi-driver/component.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awsebscsidriver
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/gohcl"
+	"github.com/kinvolk/lokomotive/pkg/components"
+	"github.com/kinvolk/lokomotive/pkg/components/util"
+)
+
+const name = "aws-ebs-csi-driver"
+
+const chartValuesTmpl = `
+enableDefaultStorageClass: {{ .EnableDefaultStorageClass }}
+`
+
+//nolint:gochecknoinits
+func init() {
+	components.Register(name, newComponent())
+}
+
+type component struct {
+	EnableDefaultStorageClass bool `hcl:"enable_default_storage_class,optional"`
+}
+
+func newComponent() *component {
+	return &component{
+		EnableDefaultStorageClass: true,
+	}
+}
+
+// LoadConfig loads the component config.
+func (c *component) LoadConfig(configBody *hcl.Body, evalContext *hcl.EvalContext) hcl.Diagnostics {
+	if configBody == nil {
+		return hcl.Diagnostics{}
+	}
+
+	return gohcl.DecodeBody(*configBody, evalContext, c)
+}
+
+// RenderManifests renders the helm chart templates with values provided.
```

Comment:
Could you please fix the capitalization here and everywhere else?

Reply:
I'm not sure what's wrong with the capitalization here - could you help me understand?

Reply:
I suspect @johananl meant https://helm.sh/docs/chart_best_practices/conventions/, even though even https://helm.sh does not respect it, as they often use `Helm Charts` term.

Reply:
I'd say
```suggestion
// RenderManifests renders the Helm chart templates with values provided.
```